packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -577,61 +578,82 @@ class SingleActivator with Diagnosticable, MenuSerializableShortcut implements S
 /// ** See code in examples/api/lib/widgets/shortcuts/character_activator.0.dart **
 /// {@end-tool}
 ///
+/// The [alt], [control], and [meta] flags represent whether the respective
+/// modifier keys should be held (true) or released (false). They default to
+/// false. [CharacterActivator] cannot check shifted keys, since the Shift key
+/// affects the resulting character, and will accept whether either of the
+/// Shift keys are pressed or not, as long as the key event produces the
+/// correct character.
+///
+/// By default, the activator is checked on all [RawKeyDownEvent] events for
+/// the [character] in combination with the requested modifier keys. If
+/// `includeRepeats` is false, only the [character] events with a false
+/// [RawKeyDownEvent.repeat] attribute will be considered.
+///
+/// {@template flutter.widgets.shortcuts.CharacterActivator.alt}
+/// On macOS and iOS, the [alt] flag indicates that the Option key (⌥) is
+/// pressed. Because the Option key affects the character generated on these
+/// platforms, it can be unintuitive to define [CharacterActivator]s for them.
+///
+/// For instance, if you want the shortcut to trigger when Option+s (⌥-s) is
+/// pressed, and what you intend is to trigger whenever the character 'ß' is
+/// produced, you would use `CharacterActivator('ß')` or
+/// `CharacterActivator('ß', alt: true)` instead of `CharacterActivator('s',
+/// alt: true)`. This is because `CharacterActivator('s', alt: true)` will
+/// never trigger, since the 's' character can't be produced when the Option
+/// key is held down.
+///
+/// If what is intended is that the shortcut is triggered when Option+s (⌥-s)
+/// is pressed, regardless of which character is produced, it is better to use
+/// [SingleActivator], as in `SingleActivator(LogicalKeyboardKey.keyS, alt:
+/// true)`.
+/// {@endtemplate}
```

Comment:
Thanks for taking the time to update these docs here.  Were you trying to handle the alt+letter accelerator shortcut with CharacterActivator?